client/client-lib/src/ln/mod.rs

```diff
@@ -156,7 +156,7 @@ impl<'c> LnClient<'c> {
     pub async fn get_offer(&self, payment_hash: Sha256Hash) -> Result<IncomingContractOffer> {
         self.context
             .api
-            .await_offer(payment_hash, Duration::from_secs(10))
+            .await_offer(payment_hash, Duration::from_secs(15))
```

Comment:
oh..I thought I changed it back correctly..Should I change it to 10 again ? (CI will fail more often then I think)

Reply:
For me the unit tests pass regardless if I set that to `1` or `30` so maybe it's unrelated?

Reply:
ok weird, I'll just set it back to 10 then